.github/docs/CustomChatDurationFeature.md

```diff
@@ -0,0 +1,27 @@
+# Custom Chat Duration
+
+Set custom total duration of the newly started chat session. This enables you to define how long your customers have to resume a chat interaction before it expires. 
+
+Learn more about chat duration: https://docs.aws.amazon.com/connect/latest/APIReference/API_StartChatContact.html#connect-StartChatContact-request-ChatDurationInMinutes
```

Comment:
Maybe inline the link. I feel like every other link is in-lined

Reply:
Updated